vendor/engines/split_accounts/spec/factories/split_accounts.rb

```diff
@@ -0,0 +1,45 @@
+FactoryGirl.define do
+  factory :split_account do
```

Comment:
Yeah, that sounds like a good idea to me, but that's fine if it's a change for later.